[2022-12-15T09:00:09.641+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: train_pos_etl.load_to_bq scheduled__2022-12-14T00:00:00+00:00 [queued]>
[2022-12-15T09:00:09.690+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: train_pos_etl.load_to_bq scheduled__2022-12-14T00:00:00+00:00 [queued]>
[2022-12-15T09:00:09.691+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-15T09:00:09.691+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-15T09:00:09.692+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-15T09:00:11.581+0000] {taskinstance.py:1304} INFO - Executing <Task(GCSToBigQueryOperator): load_to_bq> on 2022-12-14 00:00:00+00:00
[2022-12-15T09:00:11.622+0000] {standard_task_runner.py:55} INFO - Started process 12533 to run task
[2022-12-15T09:00:11.657+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'train_pos_etl', 'load_to_bq', 'scheduled__2022-12-14T00:00:00+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_train_pos.py', '--cfg-path', '/tmp/tmpqchvwu47']
[2022-12-15T09:00:11.695+0000] {standard_task_runner.py:83} INFO - Job 95: Subtask load_to_bq
[2022-12-15T09:00:12.211+0000] {task_command.py:389} INFO - Running <TaskInstance: train_pos_etl.load_to_bq scheduled__2022-12-14T00:00:00+00:00 [running]> on host 1f859f359fd5
[2022-12-15T09:00:12.443+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=noe
AIRFLOW_CTX_DAG_ID=train_pos_etl
AIRFLOW_CTX_TASK_ID=load_to_bq
AIRFLOW_CTX_EXECUTION_DATE=2022-12-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-14T00:00:00+00:00
[2022-12-15T09:00:12.475+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2022-12-15T09:00:12.520+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2022-12-15T09:00:12.523+0000] {gcs_to_bigquery.py:367} INFO - Using existing BigQuery table for storing data...
[2022-12-15T09:00:12.588+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2022-12-15T09:00:13.107+0000] {gcs_to_bigquery.py:397} INFO - Executing: {'load': {'autodetect': True, 'createDisposition': 'CREATE_IF_NEEDED', 'destinationTable': {'projectId': 'midyear-cursor-371415', 'datasetId': 'train_dataset', 'tableId': 'staging__train_pos'}, 'destinationTableProperties': {'description': None, 'labels': None}, 'sourceFormat': 'CSV', 'skipLeadingRows': 1, 'sourceUris': ['gs://etl-***-train-pos-bucket/train_pos.csv'], 'writeDisposition': 'WRITE_APPEND', 'ignoreUnknownValues': False, 'allowQuotedNewlines': False, 'encoding': 'UTF-8'}}
[2022-12-15T09:00:13.108+0000] {bigquery.py:1539} INFO - Inserting job ***_train_pos_etl_load_to_bq_2022_12_14T00_00_00_00_00_a7be98ac0a858e57d301ad293883d3ed
[2022-12-15T09:00:15.009+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/transfers/gcs_to_bigquery.py", line 456, in execute
    job.result(timeout=self.result_timeout, retry=self.result_retry)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/base.py", line 728, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/future/polling.py", line 137, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Provided Schema does not match Table midyear-cursor-371415:train_dataset.staging__train_pos. Field TrainNumber has changed type from INTEGER to STRING
[2022-12-15T09:00:15.030+0000] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=train_pos_etl, task_id=load_to_bq, execution_date=20221214T000000, start_date=20221215T090009, end_date=20221215T090015
[2022-12-15T09:00:15.057+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 95 for task load_to_bq (400 Provided Schema does not match Table midyear-cursor-371415:train_dataset.staging__train_pos. Field TrainNumber has changed type from INTEGER to STRING; 12533)
[2022-12-15T09:00:15.114+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2022-12-15T09:00:15.153+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
